docs/tutorial/part-six/index.md

```diff
@@ -169,11 +151,11 @@ export const query = graphql`
 `
 ```
 
-Now the frontpage should look like:
+Nun sollte die Startseite so aussehen:
 
 ![frontpage](frontpage.png)
 
-But your one blog post looks a bit lonely. So let's add another one at
+Aber dein einziger Blog-Beitrag sieht ein bisschen einsam aus. Also lasst uns noch einen hinzufügen bei
```

Comment:
In case you want to stay with "Sie": then this sentence should become

```suggestion
Aber Ihr einziger Blog-Beitrag sieht ein bisschen einsam aus. Also lassen Sie uns noch einen hinzufügen bei
```